### DISCUSSION TITLE
Where are the archives of the MusicXML forum?

### DISCUSSION THREAD

**Discussion by lemzwerg** [w3c-cg/musicxml#501] [General]
The MusicXML mailing list has been transferred to a MusicXML forum some time ago AFAIK, but this forum appears to be no longer available – at least I can't find it in the internet.

The only thing I was able to locate is this github discussion, but it lacks all the previous posts.

Where can I either visit the old forum or download the messages to view them offline?

**Comment by mdgood** [comment]
Unfortunately, those archives are no longer available. Some of it is available on the Wayback Machine at archive.org, but most of it is not. I have tried to update all the issues that used to have a forum link to either remove the link, or replace it with the archive.org link when it was available.

**Reply by lemzwerg** [reply]
This is most unfortunate – more than 20 years of information gone, and we are forced to reiterate all questions here, which is both tedious for the people to ask and for those who answer.  There are so many loose ends with MusicXML, and there is, besides the issues here on github, *zero* information elsewhere!

How could that happen?
